packages/client/src/components/authForm/authForm.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { ChangeEvent, useEffect, useState } from 'react';
+import style from './authForm.module.scss';
+import classNames from 'classnames';
+import { Button, ButtonVariation } from '../button/button';
+import { Link } from 'react-router-dom';
+import { getInputName } from '../../utils/helpers';
+import validate from '../../service/validate/validate';
+
+export interface IAuthFormProps {
+  submitBtnName: string;
+  children: JSX.Element | JSX.Element[];
```

Comment:
Неправильный тип children, лучше использывать `PropsWithChildren`

``
```suggestion
  export const AuthForm = (props: PropsWithChildren<IAuthFormProps>) => {
```

Reply:
Сделал